integration-tester/tester/tester.go

```diff
@@ -0,0 +1,302 @@
+package tester
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/abetterinternet/prio-server/manifest-updater/manifest"
+	"github.com/abetterinternet/prio-server/workflow-manager/kubernetes"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	LabelKey   = "type"
+	LabelValue = "integration-tester"
+)
+
+type Tester struct {
+	kubeClient         *kubernetes.Client
+	namespace          string
+	ingestorLabel      string
+	ownManifestUrl     string
+	phaManifestUrl     string
+	facilManifestUrl   string
+	serviceAccountName string
+
+	facilitatorImage string
+	pushGateway      string
+	awsAccountId     string
+}
+
+func New(
+	kubeConfigPath,
+	namespace, ingestorLabel,
+	ownManifestUrl, phaManifestUrl, facilManifestUrl,
+	serviceAccountName, facilitatorImage,
+	pushGateway, awsAccountId string,
+	dryRun bool) *Tester {
+
+	kubeClient, err := kubernetes.NewClient(namespace, kubeConfigPath, dryRun)
+
+	if err != nil {
+		log.Fatalf("error creating a new kubernetes client: %v", err)
```

Comment:
I did this but I don't think it matters much because for small micro-services (or micro-jobs? not sure what to call it) like this, if this fails, then we're going to quit anyway.